getting-started/getting-started.md

```diff
@@ -185,39 +224,55 @@ This creates a new recipe called `com.yourorg.VetToVeterinary`. Now add it to th
+
+* The source files in the `vet` package have been moved to the newly created `veterinary` package
+* References such as import statements have been updated to reflect the new name
+* All of the files have been formatted according to the previously defined AutoFormat rules
 
 ![Git diff showing updated import statements](<../.gitbook/assets/image (10).png>)
 
-You can test that spring-petclinic-migrations still builds & passes its tests by running `./mvnw clean install` or `./gradlew build`. At this point, you know how to configure and run any recipe included in Rewrite itself.
+From there, you can confirm that everything still builds and passes its tests by running `./mvnw clean install` or `./gradlew build`.
 
-## Step 5: Running Recipes from External Modules
+## Step 6: Running Recipes from External Modules
 
-Not all rewrite recipes are bundled into the core libraries. In this example, we'll use [rewrite-spring](https://github.com/openrewrite/rewrite-spring), a rewrite-maintained module that contains Spring, Mockito, JUnit, and AssertJ-related refactoring recipes.
+At this point, you know how to configure and run any recipe included in Rewrite itself. However, many recipes are not bundled into the core library. For example, all of the Spring, Mockito, JUnit, and AssertJ-related recipes maintained by the Rewrite team live in the [rewrite-spring repository](https://github.com/openrewrite/rewrite-spring).
 
-1. Add the `rewrite-recipe-bom` to manage versions of various Rewrite artifacts
-2. Add a `rewrite` dependency on rewrite-spring
-3. Set the `org.openrewrite.java.spring.boot2.SpringBoot2JUnit4to5Migration` recipe as active.
+{% hint style="info" %}
+You can search through all of the recipes in the [OpenRewrite docs](https://docs.openrewrite.org/reference/recipes). Each recipe page has instructions for how to import the recipe and what parameters (if any) need to be included. 
+{% endhint %}
 
-After applying these steps, the relevant portions of your build file will look like this:
+Let's pretend that you want to migrate JUnit 4 to JUnit 5 in a Spring project you have. If you take a look at the [Usage section](https://docs.openrewrite.org/reference/recipes/java/spring/boot2/springboot2junit4to5migration#usage) in the [JUnit 4 to 5 migration recipe](https://docs.openrewrite.org/reference/recipes/java/spring/boot2/springboot2junit4to5migration), you'll see what you need to include in your `build.gradle` or `pom.xml` file in order to use this recipe. 
+
+Below, we'll walk through the [Maven](#maven--external-modules) and [Gradle](#gradle--external-modules) changes and provide some additional context around said changes.
+
+### Maven + External Modules
+
+For Maven projects, you'll need to: 
+
+* Add the recipe to the `activeRecipes` list
+* Add a dependency on `rewrite-spring` (where the JUnit 4 to 5 recipe lives)
+* Specify a version of `rewrite-spring` to use
```

Comment:
I'm unsure how someone would know which version to specify for a given recipe. It seems like there's a possibility that people could specify the wrong version or have versions that are mismatched. Is that true? Could anyone give me some additional context on that front?

Reply:
Well, we actually specify the version in the example pom.xml. We update that version with each release.